packages/context/src/context.ts

```diff
@@ -34,6 +34,8 @@ const pEvent = require('p-event');
 
 const debug = debugModule('loopback:context');
 
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
Why are we disabling this rule instead of fixing the linting violations as we do in other files?